lib/widget/video.dart

```diff
@@ -23,42 +26,102 @@ class Video extends StatefulWidget
 
   @override
   Map<String, Function> getters() {
-    return {};
+    return {
+      'isInitialized': () => _controller._playerController?.value.isInitialized,
+      'isBuffering': () => _controller._playerController?.value.isBuffering,
+      'isLooping': () => _controller._playerController?.value.isLooping,
+      'isPlaying': () => _controller._playerController?.value.isPlaying,
+      'isEnded': () => _controller._playerController?.value.isCompleted,
+    };
   }
 
   @override
   Map<String, Function> methods() {
-    return {};
+    return {
+      'stop': () => _controller.playerCapabilities?.stop(),
+      'pause': () => _controller.playerCapabilities?.pause(),
+      'play': () => _controller.playerCapabilities?.play(),
+      'mute': () => _controller.playerCapabilities?.mute(),
+      'unmute': () => _controller.playerCapabilities?.unmute(),
+      'seekTo': (value) => _controller.playerCapabilities?.seekTo(value),
+      'setPlaybackRate': (value) =>
```

Comment:
do we need this if we already have the setter?

Reply:
Updated